docs/src/functions_list.md

```diff
@@ -4,7 +4,156 @@
 CurrentModule = SpecialFunctions
 ```
 
-```@autodocs
-Modules = [SpecialFunctions]
-Order   = [:module, :type, :function]
+
+## Gamma Function
+
+```@docs
+gamma
+loggamma
+logabsgamma
+loggamma1p
+logfactorial
+digamma
+invdigamma
+trigamma
+polygamma
+gamma_inc
+gamma_inc_inv
+loggammadiv
+gammax
+rgammax
+rgamma1pm1
+gamma_inc_asym
+gamma_inc_cf
+gamma_inc_fsum
+gamma_inc_minimax
+gamma_inc_taylor
+gamma_inc_taylor_x
+gamma_inc_temme
+gamma_inc_temme_1
+gamma_inc_inv_psmall
+gamma_inc_inv_qsmall
+gamma_inc_inv_alarge
+auxgam
+```
+
+### Beta Function
+```@docs
+beta
+logbeta
+logabsbeta
+logabsbinomial
+beta_inc
+beta_inc_inv
+ncbeta
+ncbeta_poisson
+ncbeta_tail
+beta_inc_power_series1
+beta_inc_power_series2
+beta_inc_asymptotic_asymmetric
+beta_inc_asymptotic_symmetric
+beta_inc_power_series
+beta_inc_diff
+beta_inc_cont_fraction
+beta_integrand
+```
+
+### Utilities
+
+```@docs
+chepolsum
+lambdaeta
+stirling_corr
+stirling_error
+esum
+coeff1
+coeff2
+coeff3
+ncF
+```
+
+## Exponential and Trigonometric Integrals
+
+```@docs
+expint
+expinti
+expintx
+sinint
+cosint
+```
+
+## Error Functions, Dawson’s and Fresnel Integrals
+
+```@docs
+erf
+erf(::Real, ::Real)
+erfc
+logerf
+erfcinv
+erfcx
+logerfc
+logerfcx
+erfi
+erfinv
+dawson
+faddeeva
+```
+
+## Airy and Related Functions
+
+```@docs
+airyai
+airyaiprime
+airybi
+airybiprime
+airyaix
+airyaiprimex
+airybix
+airybiprimex
+```
+
+## Bessel Functions
+
+```@docs
+besselj
+besselj0
+besselj1
+besseljx
+sphericalbesselj
+bessely
+bessely0
+bessely1
+besselyx
+sphericalbessely
+besselh
+besselhx
+hankelh1
+hankelh1x
+hankelh2
+hankelh2x
+besseli
+besselix
+besselk
+besselkx
+jinc
+```
+
+## Elliptic Integrals
+
+```@docs
+ellipk
+ellipe
+```
+
+## Zeta and Related Functions
+
+```@docs
+eta
+zeta
+```
+
+## Misc
+
+```@docs
+_cotpi
```

Comment:
This sounds like an internal function which shouldn't be exposed in public API

Reply:
Agreed. The `@autodocs` block exposed it, so I was keeping it consistent. Will remove it. Thank you!